python/lsst/ap/association/association.py

```diff
@@ -114,30 +114,39 @@ def run(self,
               not matched a new DiaSource. (`int`)
```

Comment:
Please update the docstring to describe the additional things that are returned now.

Reply:
Added